Week-3/assignment-2.js

```diff
@@ -0,0 +1,71 @@
+console.log('--------------------');
+console.log('This is assignment 2');
+
+console.log('------------Bind------------');
+
+a=10 //Global variable 
+b=10 //Global variable 
+
+// Bind
+function multiply(a,b) {
+
```

Comment:
@soumya213 multiply is a simple function which accepts a and b. If we pass different values to a & b it will give its corresponding value. Do you think bind is needed here. 

Can you define a object and have some methods, and borrow that method for a another object by binding.

Reply:
@RajeshSivanesan I have updated the Bind function methods

```
// Bind

let name1={
    name:'Soumya',
    title:'Dhar',
    
    works: function (workplace) {
        console.log(this.name + ' ' + this.title + ' is working ' + workplace);
    }
}

let name2={
    name:'Sthiti',
    title:'Nath',
}

let name3={
    name:'Abhsekh',
    title:'Panigrahi',
}

workPlace= name1.works.bind(name2,'tcs')
workPlace()

workPlace= name1.works.bind(name3,'pwc')
workPlace()

/*
 bind() method doesn’t immediately execute the function. It just returns a new version of the function whose this sets to thisArg argument. 
*/
```